### PR TITLE
Support Multi-Format Ads - Banner & Video

### PR DESCRIPTION
**Pull request by RajeshRhythmone**
## Type of change

- [x] Feature
- [x] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [x] Other

## Description of change
- Support for Multi-Format Ads - Banner & Video
- Updated ttl value for Banner & Video.

- contact email of the adapter’s maintainer: support@rhythmone.com
- [x] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- https://github.com/prebid/prebid.github.io/pull/1065

### CODE REVIEW

**Comment by mkendall07** [comment]
This pull request **fixes 1 alert** when merging a703b3d1f94eb673275a7e6c21a3e6698bfab79b into 14bf872264e6ab06c64653285cf06cbc16c76f71 - [view on LGTM.com](https://lgtm.com/projects/g/prebid/Prebid.js/rev/pr-0e238733c8813fd6e1c8bdba0a2b6f22665075ca)

**fixed alerts:**

* 1 for Useless conditional

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by jsnellbaker** [comment]
Hi @RajeshRhythmone 

I tried to use the new endpoint, however it doesn't seem to be working.  I'm still getting a 204 response despite multiple attempts.

Below is a copy of the payload/URL for reference:
`http://tag.1rx.io/rmp/80184/0/mvo?z=1r&hbv=1.38.0-pre,1.0.2`
```
{"id":"17a8cfdf77b407","imp":[{"id":"div-gpt-ad-1460505748561-0","bidfloor":0,"secure":0,"banner":{"w":300,"h":250},"ext":{"bidder":{"placementId":"411806","zone":"1r","path":"mvo"}}}],"site":{"domain":"ap.localhost","page":"http://ap.localhost:9999/integrationExamples/gpt/hello_world.html?pbjs_debug=true","ref":"http://ap.localhost:9999/integrationExamples/gpt/hello_world.html?pbjs_debug=true"},"device":{"ua":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.80 Safari/537.36","devicetype":2,"ip":"","dnt":0},"user":{"ext":{"consent":""}},"at":1,"tmax":1000,"regs":{"ext":{"gdpr":true}}}
```

File: modules/rhythmoneBidAdapter.js [hunk 1]
```diff
@@ -1,12 +1,24 @@
 'use strict';
 
+import * as utils from 'src/utils';
```
Comment (jsnellbaker) [inline modules/rhythmoneBidAdapter.js]: Due to a recent merge (see #3435), we now require any imports for a core or `modules` file (like this adapter file) to use a relative path style.

Can you please update this line here?

**Comment by mkendall07** [comment]
This pull request **fixes 1 alert** when merging dd1a71ea296a4fe36ee167c2348521944aa3beed into 92bfb2904e2633c5da3fd78386f67d82488f490a - [view on LGTM.com](https://lgtm.com/projects/g/prebid/Prebid.js/rev/pr-afab823b72a880fb7eac0e5511990d8e84adf9e2)

**fixed alerts:**

* 1 for Useless conditional

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by mkendall07** [comment]
This pull request **fixes 1 alert** when merging c74b7daad3ce1eb3c55d1b4846804d8ad4645258 into 92bfb2904e2633c5da3fd78386f67d82488f490a - [view on LGTM.com](https://lgtm.com/projects/g/prebid/Prebid.js/rev/pr-7e29c24a677287e25ca1cee1f0897d719a9e9139)

**fixed alerts:**

* 1 for Useless conditional

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by mkendall07** [comment]
This pull request **fixes 1 alert** when merging 5d2ee53429b0b0d62a84a0b6046e270fd0554d94 into 92bfb2904e2633c5da3fd78386f67d82488f490a - [view on LGTM.com](https://lgtm.com/projects/g/prebid/Prebid.js/rev/pr-24b06e9c89fd1b77147f67e5d9eca9ac270d2014)

**fixed alerts:**

* 1 for Useless conditional

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: modules/rhythmoneBidAdapter.js (hunk 1 above)
Comment (RajeshRhythmone) [reply modules/rhythmoneBidAdapter.js]: I have updated the recent merge mentioned. Please validate.
@jsnellbaker if you still see 204 No Content response, please let me know your location. 
Just want to make sure ads are targeted to your location.

**Comment by jsnellbaker** [comment]
@RajeshRhythmone Thanks for making the import updates.  In terms of the ad delivery, I'm located outside of Philadelphia.  If you need any more specifics, let me know and we can chat through e-mail.

**Comment by RajeshRhythmone** [comment]
@jsnellbaker Sure, chat through e-mail should work for me. Let me know your id, will ping you. Thanks

**Comment by jsnellbaker** [comment]
Hi @RajeshRhythmone my email is jason.snellbaker@xandr.com

**Comment by jsnellbaker** [comment]
@RajeshRhythmone Thanks for sending the customized test params.  They worked fine for me.

**Comment by RajeshRhythmone** [comment]
Thanks @jsnellbaker
There was a change request for user-facing APIs. https://github.com/prebid/prebid.github.io/pull/1065
I want to bring this to your attention as well.

**Pull request merged**